packages/tables/src/Columns/CheckboxColumn.php

```diff
@@ -49,6 +50,8 @@ public function toEmbeddedHtml(): string
         $inputAttributes = $this->getExtraInputAttributeBag()
             ->merge([
                 'disabled' => $isDisabled,
+                'wire:loading.attr' => 'disabled',
```

Comment:
Has this been tested to work with disabled columns?
I commented about that on the previous PR too since I believe that was the original challenge I couldn't work around last year.

@danharrin @thethunderturner

Reply:
I believe it works. Unless Im still missing your point... If the column is disabled, then any all will be disabled and during loading they will stay disabled. If some of them are enabled, then they will disable until they're finished loading. Is this what you mean with disabled columns?

https://github.com/user-attachments/assets/705234dd-3630-4f9d-93e6-015b43c0aae9